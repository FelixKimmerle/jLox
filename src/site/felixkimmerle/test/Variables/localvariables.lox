{
    var t = 123;
    print t;
}
print t;