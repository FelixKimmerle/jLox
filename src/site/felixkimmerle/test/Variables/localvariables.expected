123
Undefined variable 't'.
[line 5]
